force-app/main/default/classes/ChartDataProviderTest.cls

```diff
@@ -46,14 +46,14 @@ public class ChartDataProviderTest {
     public override List<ChartDataProvider.ChartData> getData() {
       final List<ChartDataProvider.ChartData> chartDatas = new List<ChartDataProvider.ChartData>();
       ChartDataProvider.ChartData aChartData = new ChartDataProvider.ChartData();
-      aChartData.label = TEST_LABEL;
+      aChartData.labels = TEST_LABEL;
       aChartData.detail = TEST_DATA;
       aChartData.bgColor = TEST_COLOR;
       chartDatas.add(aChartData);
       return chartDatas;
     }
   }
-  public static final String TEST_LABEL = 'test';
+  public static final List<String> TEST_LABEL = new List<String>{ 'test' };
```

Comment:
Small detail: what about renaming the TEST_LABEL constant to TEST_LABELS (in plural) as now it is a list instead of a String ?